client/app/src/main/AndroidManifest.xml

```diff
@@ -63,7 +63,11 @@
         </activity>
         <activity android:name=".Activity.CreateRoutePassengerActivity">
         </activity>
-        <activity android:name=".Activity.RouteActivity"
+        <activity
+            android:name=".Activity.RouteActivity"
+            android:theme="@style/AppTheme2">
+        </activity>
+        <activity android:name=".Activity.FirstTimeActivity"
             android:theme="@style/AppTheme2">
         </activity>
     </application>
```

Comment:
Control statements "if", "while" and "else if" should have one space between the control keyword and opening parenthesis according to PHP conventions.